agent/agent.go

```diff
@@ -1485,15 +1485,30 @@ func writeFileAtomic(path string, contents []byte) error {
 		return err
 	}
 	if _, err := fh.Write(contents); err != nil {
+		logger.Printf("[INFO] Writing to temp file at %v failed, deleting...\n", tempPath)
```

Comment:
never mind about the debug level, this just looks like a printf that's always logged.